src/sources/tripadvisor/convert.rs

```diff
@@ -82,11 +123,47 @@ pub fn build_poi(
                 .map(ToString::to_string)
         })
         .unwrap_or_else(|| category.clone())
+        .replace(" ", "")
         .to_lowercase();
 
+    let cuisine = property
+        .cuisine
+        .inner
+        .into_iter()
+        .map(|item| {
+            let ta_cuisine = get_local_string(&["us".to_string()], &build_i18n_property(item.name))
+                .map(ToString::to_string)
+                .unwrap();
+            println!("{}", ta_cuisine);
+            match CUISINE_CONVERTER
+                .contains_key(ta_cuisine.to_lowercase().replace(" ", "").as_str())
+            {
+                true => CUISINE_CONVERTER
+                    .get(ta_cuisine.to_lowercase().replace(" ", "").as_str())
```

Comment:
As `HashMap::get` returns an `Option<T>`, you don't need to call `contains_key` above.

My suggestion would be to use [`.unwrap_or(ta_cuisine)`](https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap_or), and assigning `ta_cuisine.to_lowercase().replace(" ", "")` to its own variable if the borrow checker is being annoying.